test/e2e/usersignup_test.go

```diff
@@ -72,11 +67,18 @@ func (s *userSignupIntegrationTest) TestAutomaticApproval() {
 				),
 		)
 		// create additional user to reach max space limits on both members
-		NewSignupRequest(s.Awaitilities).
+		_, mur2 := NewSignupRequest(s.Awaitilities).
 			Username("automatic2").
 			Email("automatic2@redhat.com").
 			RequireConditions(wait.ConditionSet(wait.Default(), wait.ApprovedAutomatically())...).
-			Execute(s.T())
+			Execute(s.T()).Resources()
+
+		require.NotEqual(t, mur1.Status.UserAccounts[0].Cluster.Name, mur2.Status.UserAccounts[0].Cluster.Name)
```

Comment:
Could you please also update the test description when you are merging tests?
For example in this case the test description is saying that it testing that when the cluster is full then new spaces are not approved but put in the wait list. However it also now tests then when one cluster if full then a new space is places in another cluster.

Just keep in mind that test execution time is one thing but we also should keep the test readable/maintainable as much as possible. And adding proper comments in the code can help with that.